backend/analytics_server/mhq/store/repos/code.py

```diff
@@ -367,6 +367,32 @@ def get_team_repos(self, team_id) -> List[OrgRepo]:
         team_repo_ids = [tr.org_repo_id for tr in team_repos]
         return self.get_repos_by_ids(team_repo_ids)
 
+    @rollback_on_exc
+    def get_prs_merged_without_review(
+        self, team_id: str, interval: Interval, pr_filter: PRFilter = None
+    ) -> List[PullRequest]:
+        AllOrg = self.get_team_repos(team_id)
+        AllOrg_ids = [tr.id for tr in AllOrg]
```

Comment:
```suggestion
        team_repos = self.get_team_repos(team_id)
        repo_ids = [team_repo.id for team_repo in team_repos]
```

Reply:
variable names should be better and follow similar patterns as other part of the codebase. Having var names like `AllOrg` and `AllOrg_ids` makes the code harder to read and do not follow either snake case or camel case. We generally follow snake case in the python backend.

Reply:
I think we can move the repo fetching part to `PullRequestAnalyticsService` and the just pass on  repo_ids to the Repo. Separates the basic query from business layer and we can also follow the same pattern as `get_prs_merged_in_interval`